Replace adviser promo in the apply section (#4409)
DFE-Digital/get-into-teaching-app: app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

  - teacher training application
external_content:
    - content/train-to-be-a-teacher/promos/apply-for-your-course
cta_adviser:
  adviser:

>     text: Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience. Chat by phone, text or email, as little or as often as you need.

Reviewer: I think we could probably make this text a bit more relevant to the page here?

>     text: Maximise your chances of submitting a successful application with the support of a dedicated adviser with years of teaching experience. Chat by phone, text or email, as little or as often as you need.

Author: ```suggestion
    text: An adviser with years of teaching experience can help you with your personal statement. Chat by phone, text or email, as little or as often as you need.
```
